chsdi/tests/integration/test_loader.py

```diff
@@ -31,3 +31,9 @@ def test_loaderjs_debug(self):
         resp.mustcontain('proj4.js')
         resp.mustcontain('EPSG21781.js')
         resp.mustcontain('EPSG2056.js')
+
```

Comment:
Add a test for the loader if version matches the reg exp but doesn t match a version we have.